.github/workflows/codeql-analysis.yml

```diff
@@ -38,11 +38,11 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v3
+      uses: actions/checkout@d0651293c4a5a52e711f25b41b05b2212f385d28
```

Comment:
Is this the commit hash?

It gives a 404.
https://github.com/actions/checkout/commit/d0651293c4a5a52e711f25b41b05b2212f385d28

v3 commit hash looks to be `2541b1294d2704b0964813337f33b291d3f8596b`. Let me know if I'm reading this right. :slightly_smiling_face: